test/parallel/test-webcrypto-random.js

```diff
@@ -47,7 +47,7 @@ for (const ctor of intTypedConstructors) {
 }
 
 {
-  const buf = new Uint16Array(10);
+  const buf = new Uint8Array(10);
```

Comment:
This test looks like it _might_ be using `Uint16Array` intentionally. There are a couple of ways to work around this issue (for example, use `buf.toString()` instead of `Buffer.from(buf).toString('hex')` or use `Buffer.from(buf.buffer)`), but it seems to me that this test is performing the exact same test as the loop above for `ctor === Uint16Array`. So I'd suggest removing this block (line 50 to 56) entirely.

Reply:
I'm not sure if I understand a potential intention here. Using `Uint16Array` in context of this method makes sense in terms of generating random utf16 strings (i.e. `String.fromCharCode(...crypto.getRandomValues(new Uint16Array(charLength)))`) but it's probably not worth testing explicitly.

Technically, this test was worse than `ctor === Uint16Array` above, as it tested only 10 bytes out of 20 generated. :)

I agree that at this point it doesn't serve any purpose. [Documentation](https://nodejs.org/api/webcrypto.html#cryptogetrandomvaluestypedarray) allows to use instances of `Buffer` and the test for it was lacking, so I guess `Buffer.alloc(10)` will make it useful.